### ISSUE TITLE
Is this still being maintained?

### ISSUE THREAD

**Issue by chriszacker** [micromodal/Micromodal#516]
Curious if this project is still being maintained. Thanks

**Comment by heyflo** [comment]
I would also love to know, this is the best modal lib out there. If you need help maintaining, I think there are many people who would gladly assist you doing so, including myself.

**Comment by timabbott** [comment]
We've also been quite happy with the library; it works really well, and we've only found one bug in pretty heavy use. It'd be nice if it were a bit more actively maintained but I'm not particularly worried about it as a problem for using it -- it's a small enough library that it would not be hard to maintain a fork if the maintainers don't get back to working on the project at some point that it needs changes.

**Comment by dkniffin** [comment]
@ghosh Are you maintaining this library anymore? If not, would you be willing to hand over maintenance to someone else? Or at least add someone else?

**Comment by ptrsmk** [comment]
Have you considered this, @ghosh ?

**Comment by ryonwhyte** [comment]
I am also curous about this @ghosh

**Comment by robinmetral** [comment]
Three years since the last commit and no repy here—if someone is interested in taking over, maybe a community fork could be a good course of action

**Comment by dkniffin** [comment]
I have just reached out to @ghosh via the email address on his personal website. Maybe that will get his attention 🤷‍♂ 

If not, I agree that a community fork would be the best option.

**Comment by mooror** [comment]
Did you get any response to that email you sent @dkniffin? This is a great little popup library, it would be a shame to let it die.

**Comment by kalpeshsingh** [comment]
Hello folks,
I'm contributor to this library and @ghosh is main person. It is sad to see that I moved on without realising impact on community. Life is occupied with one or another different things. It was a phase for me.

I shall drop a note to @ghosh and try to see what can I do here. Any ideas how can we make this library active again. 

I feel sad to not get back to all these comments.

**Comment by mooror** [comment]
Thank you @kalpeshsingh for all the contributions you have made to this project, and for your willingness to reach out to @ghosh and see how we might make this repository active again. 

I think we can all understand how busy life can get at times and how that can switch up our priorities. Glad that you are back now 🎊

**Comment by robinmetral** [comment]
This is good news @kalpeshsingh!

> Any ideas how can we make this library active again.

Some ideas to start:

- Consider whether you, @ghosh or any other maintainers have the time and energy to pick this project up. Open-source maintenance is hard and people will understand if you prefer to hand over ownership or encourage a fork.
- Start with low-hanging fruits: a number of bugs (some major) have been reported by users over the years, look into issues with the [highest number of thumbs up](https://github.com/ghosh/Micromodal/issues?q=is%3Aissue%20state%3Aopen%20sort%3Areactions-%2B1-desc) or those with the [highest number of comments](https://github.com/ghosh/Micromodal/issues?q=is%3Aissue%20state%3Aopen%20%20sort%3Acomments-desc) (heuristics for importance). There are also a number of open PRs that might fix some of these issues.

**Comment by kalpeshsingh** [comment]
Hello folks
I dropped him a message but seems like he is off the grid. No tweet in 2 years.

Good ideas @robinmetral  but I feel it is too late for me to make a come back. Even if I do, I will be too slow.

Ultimate goal is to server purpose and we all should be open for fork version. Or I can make a last attempt to connect with few of you who can help me here. Unsure about permissions to merge code though. 


Thank you all for being curious and pouring enquiries.

**Comment by ghosh** [comment]
Sorry about this folks. As is quite evident, I have not been able to summon the motivation required to maintain this anymore. 

However, it is heartening that this little project started by @kalpeshsingh  and I years ago is still used and appreciated.

The road ahead:

- I am thinking about moving this repo to it’s own organisation where it can be maintained by the community
- @dkniffin  If you are willing, I am more than happy to hand this over to you
- We need more maintainers, I am open to adding a few more people to the repo

Thank you everybody for your interest in keeping this alive

**Comment by dkniffin** [comment]
@ghosh Yes, I'd be happy to be added as a maintainer. I don't have a *ton* of free time, but I do use this library for work and I could probably convince the "powers that be" to work on this once in a while.

I think making a GH org with several maintainers is the most sustainable option though. If you want to add me as an admin on this repo now, I could probably work on setting up that organization and migrating this over to there.

**Comment by dkniffin** [comment]
@ghosh I see you transferred this repo over. Thanks for that!

The other thing I know I'll need is npm access. I just created a npm organization for `micromodal` and I invited you to it. Can you accept that invite and move the package over there? Or give me access to the package so I can do it?
